Add PHP Signed Server Request example

jmealo opened the pull request:
This PR includes a PHP class to generate a signed server request and an example of how to use it (same example as Node.js with some HTML removed for added clarity).

I tested the output against the Node.js example and given the same ``token``, ``client_id`` and ``app_secret`` it generated identical output.

jmealo commented on signing_server_examples/php/generate_signed_request.php:
```diff
@@ -0,0 +1,61 @@
+<?php
+
+require('OpenEd/SignedServerRequest.php');
```
@themightychris: For portabilities sake of the example I should leave this as ``require`` correct? If they're using an autoloader that's PSR compliant they'd want to do ``use``.

LeonidMorozov replied:
I think we can use `require_once` here to be sure there's no issues.

themightychris replied:
Someone using an autoloader will be reorganizing this anyway. `use` has nothing to do with class loading, only namespace handling.

LeonidMorozov commented on signing_server_examples/php/OpenEd/SignedServerRequest.php:
```diff
@@ -0,0 +1,46 @@
+    private $app_secret;
+    private $client_id;
+
+    public function __construct($app_secret, $client_id)
```
please, move `client_id` to the 1st place

LeonidMorozov commented on signing_server_examples/php/OpenEd/SignedServerRequest.php:
```diff
@@ -0,0 +1,46 @@
+
+class SignedServerRequest
+{
+    private $app_secret;
```
please, rename it to `client_secret` and push it below the `client_id`.

LeonidMorozov commented:
@jmealo , thank you for PR. Good job.
Please, fix small comments and I'll merge it.

jmealo commented:
@LeonidMorozov Thank you. I believe commit b12942b addresses your feedback. I also changed ``app_secret`` to ``client_secret`` in the Node.js example for consistency.
